horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/SchemaServiceImpl.java

```diff
@@ -849,7 +854,17 @@ public SchemaExport exportSchema(int id) {
    @Transactional
    @Override
    public void importSchema(ObjectNode node) {
-      SchemaExport importSchema = Util.OBJECT_MAPPER.convertValue(node, SchemaExport.class);
+      SchemaExport importSchema;
+
+      ObjectMapper customMapper = mapper.copy();
+      customMapper.enable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
```

Comment:
Can we create this additional mapper only once and then simply inject this mapper? Otherwise we are going to create `mapper.copy();` every time we need to import something

Reply:
I did think about that, but i was being lazy ;) thinking that importing schema/test is not on the hot path

Reply:
yeah, that's true.. I was thinking the same